packages/gallery/src/components/item/itemView.js

```diff
@@ -982,6 +983,26 @@ class ItemView extends GalleryComponent {
     return tabIndex;
   }
 
+  onAnchorFocus() {
+    if (
+      (isSiteMode() || isSEOMode()) &&
+      !utils.isMobile() &&
+      window.document &&
+      window.document.activeElement &&
+      this.props.shouldUseExperimentalFeature
+      ) {
+        /* Relevant only for Screen-Reader cases:
+         When we navigate on the accessibility tree, screen readers stops and focuses on the <a> tag,
+         so it will not go deeper to the item-container keyDown event */
+        const activeElement = window.document.activeElement;
+        if (activeElement === this.itemAnchor) {
+        this.itemContainer.focus();
+      } else {
```

Comment:
Your function will always return `undefined` therefor this `else` is redundant. Please remove it